noobaa_sa/account.py

```diff
@@ -79,9 +91,11 @@ def create(self, account_name, access_key, secret_key, config_root=None):
         account_data = {
             "account_name": account_name,
             "account_email": account_email,
+            "creation_date": datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
We don't need creation date since node CLI is creating that value dynamically while executing the command

Reply:
I was getting the following error without it:
```
E           noobaa_sa.exceptions.AccountCreationFailed: Creation of account failed with error {
E             "error": {
E               "code": "InvalidSchema",
E               "message": "Schema invalid, please use required properties",
E               "detail": "Error: must have required property 'creation_date'\n    at Object.validate_account_schema (/usr/local/noobaa-core/src/manage_nsfs/nsfs_schema_utils.js:37:23)\n    at add_account (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:526:23)\n    at async manage_account_operations (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:619:9)\n    at async account_management (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:391:5)\n    at async main (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:191:13)"
E             }
E           }
```

I'll check with the devs if that's the expected behavior

Reply:
On latest noobaa build, it got fixed and it uses date dynamically

Reply:
I'm not sure if it was fixed at one point, but after confirming with Romy it was determined it's a BZ on the latest build: https://bugzilla.redhat.com/show_bug.cgi?id=2260325